src/plugins/intel_cpu/src/mkldnn_plugin.cpp

```diff
@@ -415,20 +415,30 @@ static void TransformationUpToCPUSpecificOpSet(const std::shared_ptr<ngraph::Fun
     pass_config->enable<ngraph::pass::ConvertGather1ToGather7>();
     pass_config->enable<ngraph::pass::ConvertDetectionOutput1ToDetectionOutput8>();
 
+    auto defaultPrecisions = ngraph::pass::low_precision::precision_set::int8_support;
+
     if (useLpt) {
+        bool hasINT16orINT32Levels = ngraph::pass::low_precision::LowPrecision::isFQLevelsPresent(
+            nGraphFunc,
+            { ngraph::pass::low_precision::levels::int16, ngraph::pass::low_precision::levels::int16_narrow_range,
+             ngraph::pass::low_precision::levels::int32, ngraph::pass::low_precision::levels::int32_narrow_range });
+        if (hasINT16orINT32Levels) {
+            defaultPrecisions = ngraph::pass::low_precision::precision_set::int8_int16_int32_support;
+        }
+
```

Comment:
As I noticed before this part has to be moved upper. `lpt_mkldnn_plugin.cpp` (which was updated) is partial copy of original `mkldnn_plugin.cpp` (which was not updated) to demonstrate code snippets in LPT documentation. Please, update `mkldnn_plugin.cpp` first and copy code snippets to `lpt_mkldnn_plugin.cpp` after.

Reply:
done, thank you.

Reply:
done, thank you.